### ISSUE TITLE
Traceback TypeError with Ansible 2.0.0.2

### ISSUE THREAD

**Issue by jamesboswell** [Juniper/ansible-junos-stdlib#102]
Juniper.junos breaks with Ansible 2.0.0.2

Ansible:  2.0.0.2 from Hombrew (http://brew.sh) build: (https://github.com/Homebrew/homebrew/commit/81ec8e651d7d7b84b2e36717bc6294eebeb98f2b)
junos-eznc==1.3.1 (installed directly from pip)

**Error on ansible 2.0.0.2**

```python
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 2143, in <module>
    main()
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 207, in main
    module.exit_json(**m_results)
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 1743, in exit_json
    kwargs = remove_values(kwargs, self.no_log_values)
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 626, in remove_values
    return dict((k, remove_values(v, no_log_strings)) for k, v in value.items())
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 626, in <genexpr>
    return dict((k, remove_values(v, no_log_strings)) for k, v in value.items())
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 626, in remove_values
    return dict((k, remove_values(v, no_log_strings)) for k, v in value.items())
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 626, in <genexpr>
    return dict((k, remove_values(v, no_log_strings)) for k, v in value.items())
  File "/Users/test/.ansible/tmp/ansible-tmp-1454709965.92-125910454178634/junos_get_facts", line 635, in remove_values
    raise TypeError('Value of unknown type: %s, %s' % (type(value), value))
TypeError: Value of unknown type: <class 'jnpr.junos.facts.swver.version_info'>, junos.version_info(major=(11, 4), type=R, minor=5, build=7)

fatal: [<host ip removed>]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_name": "junos_get_facts"}, "parsed": false}
```

**Rollback to ansible 1.9.4 resolves issue**
```shell
# uninstall Ansible 2
brew uninstall ansible
# Homebrew install Ansible 1.9
brew tap homebrew/versions
brew install ansible19
## fix Juniper.junos role install
ansible-galaxy install Juniper.junos --force
```

**Comment by jamesboswell** [comment]
Playbook used to create traceback above.   PyEZ is fine,  as evidenced in _log_ and _savedir_ files.   Ansible 2.x is throwing the TypeError

```yaml
---
- name: Verifying NETCONF
  hosts: routers
  roles:
    - Juniper.junos
  connection: local
  tasks:

    - junos_get_facts:
          host: "{{ inventory_hostname }}"
          user: "{{ juniper_user }}"
          passwd: "{{ juniper_password }}"
          logfile: get_fact.log
          savedir: /tmp
      register: junos

    - name: version
      debug: msg="{{ junos.facts.version }}"
````

**Comment by vnitinv** [comment]
@jamesboswell We are looking into this issue.

**Comment by vnitinv** [comment]
Fixed via #103
